src/uniprotkb/components/protein-data-views/DiseaseInvolvementView.tsx

```diff
@@ -83,7 +80,10 @@ export const DiseaseInvolvementEntry: FC<DiseaseInvolvementEntryProps> = ({
   }
   return (
     <>
-      <h3>{`${diseaseId} ${acronym && `(${acronym})`}`}</h3>
+      <h3>
+        {disease?.diseaseId || 'No disease ID'}
```

Comment:
```suggestion
        {disease?.diseaseId || <em>No disease ID</em>}
```
As to have a similar approach as in https://github.com/ebi-uniprot/uniprot-website/blob/b7cae17ffb11d66e7bbafdcf54d3bf7cd169abfa/src/supporting-data/citations/components/LiteratureCitation.tsx#L268